### DISCUSSION TITLE
Is skywalking agent (java) version 8.15.0 compatible with skywalking apm version 9.3.0?

### DISCUSSION THREAD

**Discussion by DeBruyne2020** [apache/skywalking#11707] [Q&A] [answered]
I have a question ：
Can skywalking agent(java) version 8.15.0  work compatible with skywalking apm version 9.3.0  ?   
ps:  we don't use anything about istio so we don't care about functions about istio

**Comment by wu-sheng** [comment]
Ref doc, https://skywalking.apache.org/docs/main/next/en/setup/service-agent/agent-compatibility/#compatibility

Usually, the backend is compatible with old agents, but don't use the latest agent with the old backend. There is no guarantee for that.

**Reply by wu-sheng** [reply, marked as answer]
Meanwhile, you could enter this question through `ASK AI` at skywalking.apache.org

It provides nearly as same answer as I did.

<img width="628" alt="image" src="https://github.com/apache/skywalking/assets/5441976/1639b7bd-7b3f-4219-8fae-f1b1b18695b7">

**Reply by DeBruyne2020** [reply]
Thanks a lot !
